liquibase-integration-tests/src/test/java/liquibase/dbtest/pgsql/PostgreSQLIntegrationTest.java

```diff
@@ -268,6 +268,27 @@ public void testGeneratedColumn() throws Exception {
 
     }
 
+    @Test
+    public void testGeneratedClobColumn() throws Exception {
+        assumeNotNull(getDatabase());
```

Comment:
I think we can probably add a similar test to `OracleIntegrationTest` considering the differences you stated in your differences findings comment.

Reply:
Ok, I will try to do it.

Reply:
Ok, I've added a new test for Oracle. But, unfortunately, I can't build the test part, so I can't test my test :(
Please take a look at it.

Reply:
I can't test that locally either, but let's see how it goes in GH and then we can see whether it requires any change or not. 

Thanks,
Daniel.

Reply:
It seems there is something wrong with the generated SQL executed. I'll ask someone's help who can help me with oracle to review this and I'll let you know.

Reply:
Thank you!